### ISSUE TITLE
socket hang up

### ISSUE THREAD

**Issue by xianfeng95** [puppeteer/puppeteer#7342]
**Tell us about your environment:**

* Puppeteer version: 10.0.0
* Platform / OS version: linux8
* URLs (if applicable):
* Node.js version: 16

**What steps will reproduce the problem?**

_Please include code that reproduces the issue._

1.
```javascript
      _brower = await puppeteer.launch({
        headless: false,
        args: [
          "--no-sandbox",
          "--disable-setuid-sandbox",
          "--disable-web-security",
          "--disable-gpu",
          // "--disable-dev-shm-usage",
          // "--font-render-hinting=medium"
          // '--remote-debugging-port=9222',
          // '--remote-debugging-address=127.0.0.1',
        ],
        env: {
          FONTCONFIG_PATH: `${path.resolve(__dirname, "../public/font")}`,
          // LD_LIBRARY_PATH: `${path.resolve(__dirname, "../extra/usr/lib")}`,
        }
      });

    var page = _brower.newPage();
```
2. launch the program

**What happens instead?**
program crash 
error show bellow

```javascript
ErrorEvent {
  target: WebSocket {
    _events: [Object: null prototype] { open: [Function], error: [Function] },
    _eventsCount: 2,
    _maxListeners: undefined,
    _binaryType: 'nodebuffer',
    _closeCode: 1006,
    _closeFrameReceived: false,
    _closeFrameSent: false,
    _closeMessage: '',
    _closeTimer: null,
    _extensions: {},
    _protocol: '',
    _readyState: 3,
    _receiver: null,
    _sender: null,
    _socket: null,
    _bufferedAmount: 0,
    _isServer: false,
    _redirects: 0,
    _url: 'ws://127.0.0.1:35063/devtools/browser/3efd564d-bf6d-4cd9-bf79-538acc3108bc',
    _req: null,
    [Symbol(kCapture)]: false
  },
  type: 'error',
  message: 'socket hang up',
  error: Error: socket hang up
      at connResetException (node:internal/errors:683:14)
      at Socket.socketOnEnd (node:_http_client:471:23)
      at Socket.emit (node:events:377:35)
      at endReadableNT (node:internal/streams/readable:1312:12)
      at processTicksAndRejections (node:internal/process/task_queues:83:21) {
    code: 'ECONNRESET'
  }
}
/opt/tiger/edu/ark/faas_l_answer_sheet_download/util/browser.js:43
  const page = await browser.newPage();
                             ^

TypeError: Cannot read property 'newPage' of undefined
```

**Comment by WzFFzW** [comment]
same error

**Comment by haydenzhourepo** [comment]
i have the same error on macbookpro m1

**Comment by themarcel** [comment]
I fixed this downgrading to 7.0.1

**Comment by mahdi29** [comment]
Same error on docker
Alpine 3.14 
Puppeteer 10.2.0
Chromium 93
